samples/drivers/crypto/src/main.c

```diff
@@ -35,19 +35,19 @@ LOG_MODULE_REGISTER(main);
 #error "You need to enable one crypto device"
 #endif
 
-const static uint8_t key[16] __aligned(32) = {
-	0x2b, 0x7e, 0x15, 0x16, 0x28, 0xae, 0xd2, 0xa6, 0xab, 0xf7, 0x15, 0x88,
-	0x09, 0xcf, 0x4f, 0x3c
-};
+/* Some crypto drivers require IO buffers to be aligned, i.e. due to underlying DMA requirements. */
+#define IO_ALIGNMENT_BYTES 4
```

Comment:
Since the key was aligned to 32 bytes, maybe that's a requirement for some platform? It may be safer just align all buffers to the largest alignment needed:
```
#define IO_ALIGNMENT_BYTES 32
```
which will also work for anything requiring 4 byte alignment (or 8, or 16) :)

Reply:
Reading the commit message of e1e19732bcdd00b55c5a7987580011eae0b640ee, it was always meant to be 4 bytes aligned.

Would rather get this in now, and see if anyone finds an issue when testing with RC2, instead of aligning by a (most likely) way too big value.

Reply:
ok, then 4 it shall be :)